docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative use case. You will learn how to create, send, and verify events in Knative.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](./install/index.html). 
+
+## Setting up Eventing subcomponents
```

Comment:
Would Trigger setup also go under the "Eventing Subcomponents" category, since (I think) it's an Eventing subcomponent? Currently, that section's placement is between event producers and event consumers.

Maybe an easier solution would be to have a _Setting Up_ heading with Setting Up Eventing Subcomponents, Setting Up Event Consumers, Setting Up Triggers, and Setting Up Event Producers as sub-headings, but that sounds a bit redundant.

What are your thoughts?

Reply:
Those 4 subheadings in that order seem good to me (and not redundant). Please note that as a major contributor my understanding of the concepts is not necessarily what the reader of this document will understand, so feel free to use your better judgment and ignore my suggestion. :smiley: